### ISSUE TITLE
Log files don't match console

### ISSUE THREAD

**Issue by Srekel** [OurMachinery/themachinery-public#538]
At load of my plugin, I print "Gallerizer loaded", and this shows up both in the console and in the log file.

```zig
    tm_logger_api.print.?(c.tm_log_type.TM_LOG_TYPE_INFO, "Gallerizer loaded!\n");
```
However, when I press the button in my plugin, that prints a bunch of stuff to the console but none of that ends up in the log file.

```zig

    if (tm_ui_api.button.?(ui, uistyle_in, &gallerize_button)) {
        tm_logger_api.print.?(c.tm_log_type.TM_LOG_TYPE_INFO, "LOL1");
```

Said log file: [2021-04-03 20.54.29.log](https://github.com/OurMachinery/themachinery-public/files/6253252/2021-04-03.20.54.29.log)

**Comment by niklas-ourmachinery** [comment]
Does it work if you add \n to the end of the string?

**Comment by Srekel** [comment]
Nope, that doesn't seem to help.

In fact, nothing gets logged after startup. I reloaded my plugin several times and ran it, lots of stuff printed to the console, including notification about the plugin being reloaded, but the file didn't get updated.

In this case, I did stuff right before closing it down:

![image](https://user-images.githubusercontent.com/1578924/113500668-1e03f500-9520-11eb-958c-dea730f1a02a.png)
